### PR TITLE
Jamie february papercuts 2 the workflow edition

### PR DESCRIPTION
**Pull request by jamiewaese-uncharted**
Lots of beautification happening here. 

Here's a picture, which says more than I can write here:

image

### CODE REVIEW

File: packages/client/hmi-client/src/components/operator/tera-operator-inputs.vue [hunk 1]
```diff
@@ -75,4 +76,17 @@ li {
 label:not(:last-child)::after {
 	content: ', ';
 }
+
+.truncate {
+	max-width: 180px;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+}
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/operator/tera-operator-inputs.vue:85]: Food for thought: names tend to get pretty long in scientific community - I think we will need a better way of dealing with them on top of truncations.

File: packages/client/hmi-client/src/components/model/petrinet/model-diagrams/tera-model-diagram.vue [hunk 2]
```diff
@@ -253,6 +253,8 @@ main {
 :deep(.graph-element svg) {
 	width: 100%;
 	height: 100%;
+	background: var(--gray-50) !important;
+	border: none !important;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/model/petrinet/model-diagrams/tera-model-diagram.vue:257]: 😱

File: packages/client/hmi-client/src/components/operator/tera-operator-header.vue [hunk 3]
```diff
@@ -77,6 +77,9 @@ header {
 	transition: all 80ms ease;
 }
 
+.header-label {
+	font-size: 0.85rem;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/operator/tera-operator-header.vue:81]:
```suggestion
	font-size: var(--font-caption);
```

File: packages/client/hmi-client/src/components/operator/tera-operator-title.vue [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
 
 <style scoped>
-h5 {
+h6 {
 	display: -webkit-box;
 	-webkit-box-orient: vertical;
 	text-overflow: ellipsis;
 	overflow: hidden;
 	-webkit-line-clamp: 3;
+	font-weight: 600;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/operator/tera-operator-title.vue:12]:
```suggestion
	font-weight: var(--font-weight-semibold);
```

File: packages/client/hmi-client/src/workflow/tera-workflow.vue [hunk 5]
```diff
@@ -815,6 +838,6 @@ function resetZoom() {
 	display: flex;
 	align-items: center;
 	flex-direction: row;
-	gap: 1rem;
+	gap: 0.5rem;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/workflow/tera-workflow.vue:841]:
```suggestion
	gap: var(--gap-small);
```

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/components/operator/tera-operator-inputs.vue (hunk 1 above)
Comment (jamiewaese-uncharted) [reply packages/client/hmi-client/src/components/operator/tera-operator-inputs.vue:85]: Agree. I was hoping to use my "truncate after 3 lines" trick, but it changes the positions of the ports below and the connection links aren't aware of the new port locations. A one line truncate is better than before, where the text spilled out of the container! Maybe we can do this one together at the hackathon?

Comment (mwdchang) [reply packages/client/hmi-client/src/components/operator/tera-operator-inputs.vue:85]: Sure, lets chat in the hackathon.

**Pull request merged**